src/opentimelineio/serialization.cpp

```diff
@@ -1251,6 +1251,61 @@ serialize_json_to_string(
     return std::string(output_string_buffer.GetString());
 }
 
+// to json_string
+std::string
+serialize_json_to_string_compact(
+    const any&                value,
+    const schema_version_map* schema_version_targets,
+    ErrorStatus*              error_status,
+    int                       indent)
```

Comment:
in the compact form, is `indent` the number of spaces between things?  or is that unused?

Reply:
It's unused. I forgot to remove it.